src/containers/AccountConnection.jsx

```diff
@@ -220,7 +216,10 @@ class AccountConnection extends Component {
 
   render () {
     const { t, existingAccount, connector, fields } = this.props
-    const { submitting, deleting, error, credentialsError, isSuccessTimedOut } = this.state
+    const { submitting, deleting, error, isSuccessTimedOut } = this.state
+    const hasGlobalError = error && error.message && ![
```

Comment:
It was me who removed an equivalent test on `error.message` to introduce the `credentialsError` property.
I did it because I deeply believe that a variable should have only one purpose. In the way we code, our `error.message` property is used to provide a key for translation. I think that testing this key to determine the error type is an anti-pattern, because the property has now two purposes : provide a key for translation AND indicates that the error is a credential error or not. So, if one of this two purpose needs to evolve, it could immediately introduce regression.

If we really want to avoid passing a `credentialsError` property, I suggest to use another way than `error.message` to determine the type of error. I suggest for example `error.type` or `error.code`. This property has to be manually added when a credential error occurs.